jupyterhub/app.py

```diff
@@ -2144,6 +2159,7 @@ def init_tornado_settings(self):
             trusted_alt_names=self.trusted_alt_names,
             shutdown_on_logout=self.shutdown_on_logout,
             eventlog=self.eventlog,
+            app=self
```

Comment:
`pre-commit run --all-files` to appease the autoformatter gods

Reply:
Done I think